docs/bundles/SyliusFixturesBundle/custom_fixture.rst

```diff
@@ -0,0 +1,128 @@
+Custom fixture
+==============
+
+Basic fixture
+-------------
+
+Let's create a fixture that loads all countries from ``Intl`` library. We'll extend on ``AbstractFixture`` in order
+to skip the configuration part for now:
+
+.. code-block:: php
+
+    namespace AppBundle\Fixture;
+
+    use Sylius\Bundle\FixturesBundle\Fixture\AbstractFixture;
+    use Sylius\Bundle\FixturesBundle\Fixture\FixtureInterface;
+
+    final class CountryFixture extends AbstractFixture implements FixtureInterface
+    {
+        private $countryManager;
+
+        public function __construct(ObjectManager $countryManager)
+        {
+            $this->countryManager = $countryManager;
+        }
+
+        public function getName()
+        {
+            return 'country';
+        }
+
+        public function load(array $options)
+        {
+            $countriesCodes = array_keys(\Intl::getRegionBundle()->getCountryNames());
+
+            foreach ($countriesCodes as $countryCode) {
+                $country = new Country($countryCode);
+
+                $this->countryManager->persist($country);
+            }
+
+            $this->countryManager->flush();
+        }
+    }
```

Comment:
Not really, it will make listings unnecessarily longer and won't make anything clearer as those methods receives only typehinted parameters and usually does not return anything (except `getName()` which is quite straightforward).